docusaurus/docs/apm.md

```diff
@@ -0,0 +1,28 @@
+---
+id: apm
+title: @idearium/apm
+---
+
+Defaults for our Elastic APM integration.
+
+## Installation
+
+```shell
+$ yarn add -E @idearium/apm
+```
+
+### Beta installation
+
+If you need to install a beta version, you can:
+
+```shell
+$ yarn add -E @idearium/apm@beta
+```
+
+## Usage
+
+To use `@idearium/apm`, simply require it at the top of your server.js file.
+
+## Advanced usage
```

Comment:
I don't think this is advanced usage? Doesn't the package require certain environment variables to exist?